openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -457,10 +457,15 @@ def get_all_courses(cls, org=None):
         """
         # Note: If a newly created course is not returned in this QueryList,
         # make sure the "publish" signal was emitted when the course was
-        # created.  For tests using CourseFactory, use emit_signals=True.
+        # created. For tests using CourseFactory, use emit_signals=True.
         course_overviews = CourseOverview.objects.all()
+
         if org:
-            course_overviews = course_overviews.filter(org=org)
+            # In rare cases, courses belonging to the same org may be accidentally assigned
```

Comment:
@nasthagiri I do have such a unit test, and it's defined at the `course_overviews` layer: https://github.com/edx/edx-platform/pull/10960/files#diff-a3131999c2ecae3cdae0e48855d41aedR483